docs/_config.yml

```diff
@@ -6,15 +6,15 @@ kramdown:
 
 # For release candidates and release branches, this should match the pom.xml Alluxio version, e.g. "1.5.0" or "1.5.0-RC1"
 # For master branch, this should be the latest released version, e.g. "1.5.0"
-ALLUXIO_RELEASED_VERSION: 1.6.1
+ALLUXIO_RELEASED_VERSION: 1.7.0-RC1
 # This should be the same as ALLUXIO_RELEASED_VERSION, except on master branch, where it should be "master"
-ALLUXIO_MASTER_VERSION_SHORT: master
+ALLUXIO_MASTER_VERSION_SHORT: 1.7.0-RC1
```

Comment:
Should these be changed?

Reply:
Yes, created https://github.com/Alluxio/alluxio/pull/6727